test/test.js

```diff
@@ -2355,21 +2355,27 @@ describe('Page', function() {
 
     it('should work with a url', SX(async function() {
       await page.goto(EMPTY_PAGE);
-      await page.addScriptTag({ url: '/injectedfile.js' });
+      const scriptHandle = await page.addScriptTag({ url: '/injectedfile.js' });
+      expect(scriptHandle.asElement()).not.toBeNull();
       expect(await page.evaluate(() => __injected)).toBe(42);
+      scriptHandle.dispose();
```

Comment:
(here and in other tests): This should be awaited so that this doesn't race with the page been closed after the test.

However, I wouldn't bother disposing these handles: since page is closed after every test, all handles are automatically released.